tests/integration/components/paper-item-test.js

```diff
@@ -0,0 +1,182 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from 'ember';
+import wait from 'ember-test-helpers/wait';
+
+const { set, get } = Ember;
+
+moduleForComponent('paper-item', 'Integration | Component | paper item', {
+  integration: true
+});
+
+test('single action checkboxes should react to checkbox clicks', function(assert) {
+  assert.expect(1);
+  set(this, 'checkboxEnabled', false);
+  this.render(hbs`
+    {{#paper-list}}
+      {{#paper-item}}
+        <p>Checkbox 1</p>
+        {{paper-checkbox class="md-secondary" value=checkboxEnabled onChange=(action (mut checkboxEnabled))}}
+      {{/paper-item}}
+    {{/paper-list}}
+  `);
+  return wait().then(() => {
```

Comment:
I think it would be worth to get rid of the `run.scheduleOnce('afterRender')`. I'm pretty sure we can get around without it.
Can you point the source of the deprecation you posted?
Maybe this: https://github.com/miguelcobain/ember-paper/blob/master/addon%2Fmixins%2Fproxy-mixin.js#L9
plus this: https://github.com/miguelcobain/ember-paper/blob/master/addon%2Fcomponents%2Fpaper-item.js#L34 (and respective classNameBinding)